docs/user_guide/connect-sdk.md

```diff
@@ -0,0 +1,130 @@
+---
+id: connect_sdk
+title: Connect your SDK
+---
+
+In order to connect your application to Unleash you need to use a client SDK for your programming language. Unleash-hosted instances will always be a protected instance, you will therefore have to specify a client secret as the authorization header when you are connecting your client SDK.
+
+On this page you find examples for connecting your application to the demo instance. If you are connecting to your own private instance you will have to remember to replace the client secret and the API url given in the examples.
```

Comment:
```suggestion
In order to connect your application to Unleash you need to use a client SDK (software developer kit) for your programming language. The SDK will handle connecting to the unleash server instance and retrieve feature toggles based upon your configuration. Both open source and the unleash enterprise offering utilize the same set of client SDKs. 

If you are an Unleash customer, our hosted instances will always be a protected instance, you will therefore have to specify a client secret as the authorization header when you are connecting your client SDK, which you can generate through the admin panel of your hosted instance. See [API Access](https://getunleash.ai/docs/user_guide/api_access) for more information.

On this page you find examples for connecting your application to the demo instance. If you are connecting to your own private instance you will have to remember to replace the client secret and the API url given in the examples.
```

Suggestion to make it clearer that SDKs are the same for both open source and unleash hosted customers, and link to the API Access documentation.